src/sentry/static/sentry/app/views/settings/components/forms/form.tsx

```diff
@@ -41,6 +41,8 @@ type Props = {
   // Save field on control blur
   saveOnBlur?: boolean;
   model?: FormModel;
+  // if set to true, preventDefault is not called
+  allowDefault?: boolean;
```

Comment:
How about `preventPreventDefault` (j/k) - I think having `preventDefault` in the name would allow people to better associate what this prop would do. Maybe something like `skipPreventDefault`?